tests/acceptance/ember-i18n-fetch-translations-test.js

```diff
@@ -1,12 +1,12 @@
-import { test } from 'qunit';
-import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
 
-moduleForAcceptance('Acceptance | ember i18n fetch translations');
+module('Acceptance | ember i18n fetch translations', function(hooks) {
+    setupApplicationTest(hooks);
 
-test('visiting /ember-i18n-fetch-translations', function(assert) {
-  visit('/');
-
-  andThen(function() {
-    assert.equal(find('div[data-illumination]').text(), 'Blinkenlichts');
-  });
+    test('visiting /ember-i18n-fetch-translations', async function(assert) {
+        await visit('/');
+        assert.equal(this.element.querySelector('div[data-illumination]').textContent, 'Blinkenlichts');
```

Comment:
Doesn't `find` exist anymore?
Did you try 
```javascript
import { visit, find } from '@ember/test-helpers';
```

Reply:
I didn't - and could!

Reply:
```
not ok 5 Chrome 64.0 - Acceptance | ember i18n fetch translations: visiting /ember-i18n-fetch-translations
    ---
        actual: >
            null
        stack: >
            TypeError: (0 , _testHelpers.find) is not a function
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:11:48)
                at <anonymous>
        message: >
            Promise rejected during "visiting /ember-i18n-fetch-translations": (0 , _testHelpers.find) is not a function
        Log: |
    ...```

Reply:
Is this because I'm defining an _asynchronous_ test?